Libs/vtkSegmentationCore/vtkSegment.h

```diff
@@ -47,6 +47,12 @@ class vtkSegmentationCore_EXPORT vtkSegment : public vtkObject
   typedef std::map<std::string, vtkSmartPointer<vtkDataObject> > RepresentationMap;
 
 public:
+  enum
+  {
+    /// Invoked when content of the master representation in a segment is changed.
+    RepresentationObjectChanged = 72100,
```

Comment:
Is it considered an abstraction violation for the vtkSegment to include vtkSegmentation?

Reply:
There have been so many different kind of modifications that can be done in vtkSegment and so far a single modified event was sufficient. It would be nice if you could check how difficult/complicated would be to do without a special modified event for representation object change (and just use the simple Modified event).